src/client/pages/signUp/SignUp.jsx

```diff
@@ -23,8 +27,8 @@ const SignUp = () => {
             body: JSON.stringify(userDetails)
         })
         const data = await res.json()
```

Comment:
Is it possible that `data` does not include a token? If so, could you do something like:

```
if (data.hasOwnProperty('token') {
  return true;
} else {
  return false
}  
```
...maybe?